examples/basics/data_rows.ipynb

```diff
@@ -2,18 +2,22 @@
   "cells": [
```

Comment:
<p>Could we update the row_data to use a labelbox dataset images, we know that picsum api is flaky and causes issues in the labeling editor. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>